src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyISQLServerBulkRecordTest.java

```diff
@@ -150,6 +150,37 @@ public void testBulkCopyDateTimePrecision() throws SQLException {
         }
     }
 
+    @Test
+    public void testBulkCopyJSON() throws SQLException {

```

Comment:
Before this merges, just a heads up and reminder that both internal and public pipelines will need the new SQL Server version that supports JSON.

Reply:
As discussed with the .Net driver team too, we need to tag all JSON unit test cases and skip them in our pipelines until SQL Server with JSON support is available. If required, we can manually run those JSON JUnit test cases.